### ISSUE TITLE
Only localhost?

### ISSUE THREAD

**Issue by orboan** [evanlucas/learnyoumongo#48]
Hello,

this is just a simple question.
To run this workshop, mongodb must be installed in the localhost (same host as the workshop) or can be used a mongodb installation from another host? I'm planning to run this workshop in a node based container and use a mongodb docker image to run a container with mongodb in the same network.
Thanks.

**Comment by JoeDoyle23** [comment]
Currently, it is hard coded to have MongoDB running on localhost with the default port of 27017. I'd be up for adding a way to change that since you're probably not the first person to want to use an existing MongoDB instance, and probably not the last person to want to use Docker!

My initial thought would be to use an environment variable with the MongoDB connection string to override the default of localhost. Would that work for you? I'm also open to other suggestions if you have any.

**Comment by orboan** [comment]
Yes, that should work as environment variables are easy to manage with Docker. If you could implement this, that would be fantastic.Thanks.

**Comment by JeffreyJamesAttenti** [comment]
I would like this implemented as well. I run mongo in containers and can't use this workshop with my setup.
